hydromt_fiat/api/hydromt_fiat_vm.py

```diff
@@ -140,6 +140,47 @@ def run_hydromt_fiat(self):
         region = self.data_catalog.get_geodataframe("area_of_interest")
         self.fiat_model.build(region={"geom": region}, opt=config_yaml.dict())
 
+        # Update exposure dataframe
+        buildings_gdf, roads_gdf = self.update_exposure_db(config_yaml)
+        return buildings_gdf, roads_gdf
+
+    def update_model(self, parameter):
```

Comment:
maybe parameter can be a list instead, and in the if statements you check if the name is in the list, so you can do multiple updates at once?